modules/creating-serverless-apps-yaml.adoc

```diff
@@ -6,10 +6,14 @@
 [id="creating-serverless-apps-yaml_{context}"]
 = Creating serverless applications using YAML
 
-To create a serverless application by using YAML, you must create a YAML file that defines a `Service` object, then apply it by using `oc apply`.
+Creating Knative resources by using YAML files uses a declarative API, which enables you to describe applications declaratively and in a reproducible manner. To create a serverless application by using YAML, you must create a YAML file that defines a Knative `Service` object, then apply it by using `oc apply`.
```

Comment:
Should we say "To create a serverless application by using YAML file. . . "

Reply:
I think it's OK as is for now, since the next line says "you must create a YAML file"?